config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "/projects", to: "projects#index"
```

Comment:
I don't think any of the US required a projects index. We don't want to be creating routes that we do not need.